### ISSUE TITLE
Export PDF twice error

### ISSUE THREAD

**Issue by huangxuewu** [amcharts/amcharts4#901]
The second export will popup an error message from Export.js?7e9c:2720

Uncaught (in promise) TypeError: Cannot set property vfs of [object Module] which has only a getter
    at Export.eval (Export.js?7e9c:2720)
    at step (tslib.es6.js?9ab4:97)
    at Object.eval [as next] (tslib.es6.js?9ab4:78)
    at fulfilled (tslib.es6.js?9ab4:68)

**Comment by xorspark** [comment]
Would it be possible to provide a demo that reproduces this or more details? I'm not seeing this error on my end when trying to export the chart as a PDF multiple times from the menu.

**Comment by huangxuewu** [comment]
Hi, i tried to reproduces this error on codesanbox but it seems the bug only exists in my environment. I will just disable the pdf export option. Thanks

**Comment by hazzo** [comment]
Hi I'm seeing this error too. I on version 4.1.4 and using it inside a Vuejs app.
First PDF download works great, rest of PDF downloads will trigger the same error:
```
Export.js?7e9c:2712 Uncaught (in promise) TypeError: Cannot set property vfs of [object Module] which has only a getter
    at Export.eval (Export.js?7e9c:2712)
    at step (tslib.es6.js?9ab4:97)
    at Object.eval [as next] (tslib.es6.js?9ab4:78)
    at fulfilled (tslib.es6.js?9ab4:68)
```

I can't provide a demo because the app is in a protected environment.
Thnks!

**Comment by martinko94** [comment]
Hi, also got the same error as @hazzo and @huangxuewu when exporting using chart export API, as specified in the docs, `chart.exporting.export('pdf')` when called more than once breaks with this err message. Only saw the error for pdf. 

```TypeError: Cannot set property vfs of [object Module] which has only a getter
    at Export.<anonymous> (Export.js:2569)
    at step (tslib.es6.js:97)
    at Object.next (tslib.es6.js:78)
    at fulfilled (tslib.es6.js:68)```

Thanks for looking at it.

**Comment by darlesson** [comment]
https://github.com/bpampuch/pdfmake/issues/1459 provided a possible clue how this error might be happening. Make sure the [amCharts dependencies](https://github.com/amcharts/amcharts4/tree/master/dist/script/deps) are not being minified. That could cause the original reference to an object to be incorrect.

**Comment by martynasma** [comment]
I'm afraid we weren't able to reproduce this.

If someone has any suggestions on how we can reproduce this error in "lab conditions", please let us know :)

Gonna wait a while before closing this.

**Comment by douglb** [comment]
I believe this might be fixed after version 4.1.8.  I was on 4.0.21 and seeing this error.  Once I upgraded to version 4.4.1, it no longer tries to load pdfmake a second time and uses the cached version.  It looks like 4.1.8 changed how this was working.
